### ISSUE TITLE
numberboxes popup

### ISSUE THREAD

**Issue by umlaeute** [chr15m/PdDroidParty#35]
the popup window for editing the numbers in a numberbox is really annoying, as it totally breaks my flow when interacting with PdDroidParty.

it would be nice if there was a "dragable" numberbox as an alternative.

**Comment by chr15m** [comment]
Agree. [PdParty](https://github.com/danomatika/PdParty) does this and it's a great feature.

**Comment by chr15m** [comment]
Alternative/additional implementation: tap the numberbox to get a cursor inside it, and edit directly.

**Comment by umlaeute** [comment]
ah well. most of the PdDroidParty user interface is **tapping** and **dragging**.
i find the switch to **typing** disruptive.

i'm not much into UX, but my mobile phone at least gives me a draggable numberbox , which i *then* can tap if i want to enter the numbers manually.

**Comment by chr15m** [comment]
> draggable numberbox , which i then can tap if i want to enter the numbers manually

:+1:

**Comment by umlaeute** [comment]
>  but my mobile phone at least gives me ...

just to clarify.
my mobile phone gives me this when using something like a date-selector.
it does not give me this with PdDroidParty.

but i would like it, if it would give this beaviour with PdDroidParty as well...

**Comment by chr15m** [comment]
@umlaeute cool thanks, that's what I understood. Drag to quickly modify, tap to enter numbers in the UI.
